### PR TITLE
Create file Emm-aan1.json

### PR DESCRIPTION
**Pull request by Emm-aan1**
renamed the file to match my account name

### CODE REVIEW

**Comment by Emm-aan1** [comment]
My pull requests are not successful and u don't know why. Please help

File: Submissions/Emm-aan1.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "website": "https://emm-aan1.github.io/mine",
+    "linkedin": "https://www.linkedin.com/in/ifeanyi-chidoka-58a02614a/",
+    "github": "https://github.com/Emm-aan1"
+  },
+  "jobTitle": "Frontend Developer",
+  "location": {
+    "city": "Lagos",
+    "state": "Delta",
+    "country": "Nigeria"
+  }
```
Comment (ProsperousHeart) [inline Submissions/Emm-aan1.json:15]:
```suggestion
  }
}

```

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Aside from missing the last `}` I believe this is ok. Please double check the main repo README.

**Review by ProsperousHeart — APPROVED** [review]
Approved!

**Pull request merged**